app/models/event.rb

```diff
@@ -0,0 +1,7 @@
+class Event < ActiveRecord::Base
+  #return an array that displays all those in this event
+  def all_swimmers
+    x = self.id
+    return Swimmer.where({"event_id" => x})
```

Comment:
Does `return Swimmer.where({"event_id" => x})` work? Or is this still a WIP?

It doesn't seem like it should work, because there is no `event_id` column in the `swimmers` table.